python/iot3/src/iot3/mobility/__init__.py

```diff
@@ -2,3 +2,33 @@
 # SPDX-FileCopyrightText: Copyright (c) 2024 Orange
 # SPDX-License-Identifier: MIT
 # Author: Yann E. MORIN <yann.morin@orange.com>
+
+import json
+
+from . import etsi
+from .gnss import GNSSReport
+
+
+def message_from_json(
+    *,
+    msg_json: str,
+) -> etsi.Message:
+    """Create a new ITS message from a json sentence"""
+    _MSG_TYPE_CLASS = dict()
+    # We should validate msg_json here before loading it,
+    # but we need to know the type of message first
+    try:
+        msg_py_obj = json.loads(msg_json)
+    except json.decoder.JSONDecodeError:
+        raise RuntimeError("Not a known ITS message")
```

Comment:
> Shouldn't the error yield about malformed JSON instead of not known ITS message ?

From the caller's perspective, it does not matter why the blob could not be made into an ITS message; what matters is just that it is not a valid ITS message, whatever the reason.

Furthemore, the `json.decoder.JSONDecodeError` would get raised for any blurb that is not even trying to be JSON data; if an arbitrary binary blob is passed, for example, that would also yield that same exception.

So, from the caller's perspective, an invalid, malformed JSON blob is just not a known ITS message, no more no less than any other arbitrary content would be.